golang-github-osbuild-composer.spec

```diff
@@ -100,16 +108,17 @@ export GOPATH=$PWD/_build:%{gopath}
 %license LICENSE
 %doc README.md
 %{_libexecdir}/osbuild-composer/
+%{_datadir}/osbuild-composer/
 %{_unitdir}/*.{service,socket}
 %{_sysusersdir}/osbuild-composer.conf
 
-%changelog
```

Comment:
Please, don't kill the changelog. :)

Reply:
Hehe :) What's the use of maintaining a changelog upstream if most entries are `New upstream release`?

Reply:
Please kill the changelog :D